servers/audio/effects/audio_stream_generator.cpp

```diff
@@ -181,7 +184,7 @@ void AudioStreamGeneratorPlayback::stop() {
 }
 
 bool AudioStreamGeneratorPlayback::is_playing() const {
-	return active; //always playing, can't be stopped
```

Comment:
This comment seemed to be outdated. Or, in fact slightly ironic, because in Godot 4.x the generator playbacks get stopped all the time :wink: